express-post-json/server.js

```diff
@@ -0,0 +1,30 @@
+import express from 'express';
+
+const app = express();
+const jsonParser = express.json();
+
+let nextId = 1;
+
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const json = [];
+  for (const key in grades) {
+    json.push(grades[key]);
+  }
+  res.json(json);
+});
+
+app.use(jsonParser);
```

Comment:
Typically, this would go at the top since it can be applied to every route. It definitely needs to go before those that require JSON, but since it does nothing if the body doesn't contain JSON (or there is no body), then it's safe to use on every route. Putting it at the top organizes the code a little better.